api/include/opentelemetry/trace/span_context.h

```diff
@@ -33,10 +33,6 @@ namespace trace_api = opentelemetry::trace;
 class SpanContext final
 {
 public:
-  // An invalid SpanContext.
-  SpanContext() noexcept
```

Comment:
Let's use `SpanContext::GetInvalid()` as the default way to create an invalid `SpanContext`. Having both `SpanContext()` and `SpanContext::GetInvalid()` returning an invalid `SpanContext` is somewhat confusing.

Reply:
Explicit delete as `SpanContext() = delete;`?

Reply:
Good call, I made it explicit.